packages/blockchain-info-components/package.json

```diff
@@ -54,12 +56,14 @@
     }
   },
   "dependencies": {
+    "@storybook/storybook-deployer": "^2.3.0",
```

Comment:
can this be installed in the top-level `package.json` with the other storybook deps? If not, this should at the very least be in `devDependencies` instead.

Reply:
Don't think it can be. i can move to dev dependencies